src/platform/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_subdirectory(cmsis)
+add_subdirectory(drivers)
+add_subdirectory(hal)
+#add_subdirectory(main)
```

Comment:
Why has this been commented? Not too clear.

Reply:
Line 4.

Reply:
Primarily because I didn't add the corresponding CMakeLists.txt in src/platform/main yet

Reply:
Gotcha.